### PR TITLE
WEB-1737 flexible fares filter

### PR DESCRIPTION
**Pull request by kengste**
WEB-1737 flexible fares filter
https://wegomushi.atlassian.net/browse/WEB-1737

### CODE REVIEW

File: src/flight-search/filtering.js [hunk 1]
```diff
@@ -5,6 +5,19 @@ function filterByPrice(trip, priceRange) {
   return trip.fares[0] && utils.filterByRange(trip.fares[0].price.amountUsd, priceRange);
 }
 
+function filterByFlexibleTickets(trip, flexible = []) {
+  const isRefundableFlag = flexible.map(str => str.toLowerCase()).indexOf("refundable") !== -1;
```
Comment (redhae) [inline src/flight-search/filtering.js:9]: can we use Array.includes() here instead?

Comment (kengste) [reply src/flight-search/filtering.js]: updated

File: src/flight-search/filtering.js [hunk 2]
```diff
@@ -5,6 +5,19 @@ function filterByPrice(trip, priceRange) {
   return trip.fares[0] && utils.filterByRange(trip.fares[0].price.amountUsd, priceRange);
 }
 
+function filterByFlexibleTickets(trip, flexible = []) {
+  const isRefundableFlag = flexible.map(str => str.toLowerCase()).includes("refundable");
```
Comment (redhae) [inline src/flight-search/filtering.js:9]: can just do `flexible.includes("refundable")` (i'm assuming the value is already lowercased tho)

Comment (kengste) [reply src/flight-search/filtering.js]: updated

**Review by redhae — APPROVED** [review]

**Pull request merged**